handlers/fantasy.js

```diff
@@ -1,7 +1,68 @@
 var Dota2 = require("../index"),
-    util = require("util");
+    util = require("util"),
+    Long = require('long');
 
 // Methods
+
+// Returns a list of promises which resolve to a list of players, with player stats if the call to the GC was succesful.
+// Maximum delay before promises resolve is ~20s. 
+// A player object looks like this:
+// {
+//     'account_id' : uint32,
+//     'cards' : [{'id' : uint32, 'bonuses' : uint64}],
+//     'stats' : CMsgGCToClientPlayerStatsResponse (if GC responded before 2s timeout)
+// }
+// For magic numbers, see -> https://raw.githubusercontent.com/SteamDatabase/GameTracking/master/dota/game/dota/pak01_dir/scripts/items/items_game.txt
+Dota2.Dota2Client.prototype.requestPlayerCardsByPlayer = function() {
+    if(this.Inventory) {
+        var playercards = this.Inventory.filter(item => item.def_index == 11953);
+        var promises = [];
+        // Sort cards per player
+        var players = playercards.reduce((players, card)=>{
+            var id_attr = card.attribute.filter(attr => attr.def_index == 424)[0];
+            var account_id = Buffer.from(id_attr.value_bytes.buffer).readUInt32LE(id_attr.value_bytes.offset);
```

Comment:
Oops! I forgot `.offset` as a parameter for `readUInt32`. If you call `requestPlayerCardsByPlayer` multiple times without offset it will be increased by 4 every time.
`id_attr.value_bytes.readUInt32(id_attr.value_bytes.offset)` - I don't know how to make it better.